src/fmu/dataio/dataio.py

```diff
@@ -507,14 +493,14 @@ def _show_deprecations_or_notimplemented(self) -> None:
             warn(
                 "The 'runpath' key has currently no function. It will be evaluated for "
                 "removal in fmu-dataio version 2. Use 'casepath' instead!",
-                PendingDeprecationWarning,
+                UserWarning,
             )
 
         if self.grid_model:
             warn(
                 "The 'grid_model' key has currently no function. It will be evaluated "
                 "for removal in fmu-dataio version 2.",
-                PendingDeprecationWarning,
+                UserWarning,
```

Comment:
Do we know that `UserWarning` is visible to the user...? Ref discussions

Reply:
After chatting with chatGPT 😄  I believe it will

> Yes, if you have a warnings.warn("message", UserWarning) call in the __init__ method of your class, and a user imports your class into their Python script and creates an instance of the class, they will see the warning when they run the script, unless they have specifically configured the warnings system to suppress it.

Reply:
OK :) We can also confirm once this is deployed to RMS, I guess